services/management/dbaas/components_service.go

```diff
@@ -125,23 +126,26 @@ func (c componentsService) versions(ctx context.Context, params componentsParams
 	return versions, nil
 }
 
-func (c componentsService) matrix(m map[string]component) map[string]*dbaasv1beta1.Component {
+func (c componentsService) matrix(m map[string]component, minimalVersion ...*goversion.Version) map[string]*dbaasv1beta1.Component {
```

Comment:
I would suggest to do this
```suggestion
func (c componentsService) matrix(m map[string]component, minimalVersion *goversion.Version) map[string]*dbaasv1beta1.Component {
```

and below just check if it's nil or not

Reply:
Right, I changed it.